api/envoy/admin/v2alpha/certs.proto

```diff
@@ -33,6 +35,12 @@ message CertificateDetails {
 
   // Minimum of days until expiration of certificate and it's chain.
   uint64 days_until_expiration = 4;
+
+  // Indicates the time from which the certificate is valid.
+  google.protobuf.Timestamp valid_from = 5;
```

Comment:
Sorry if this is totally naive, but can't we infer this information directly from the cert with BoringSSL library calls?

Reply:
@htuch Yeah, there is no direct way to get it from BoringSSL calls.

Reply:
Surprising :) When I run `openssl` on the CLI against a cert, I can see information such as expiration date, so I was assuming there would be some programmatic access to this. @PiotrSikora do you know if there's any way to do this?

Reply:
Oh, I see, this is a summary we're providing as a convenience. Sure, OK, this is fine then.